spacemacs/packages.el

```diff
@@ -3708,10 +3708,15 @@ one of `l' or `r'."
               (,(listify-key-sequence
                  (kbd dotspacemacs-emacs-leader-key)) . "Spacemacs root")))
       (nconc which-key-prefix-title-alist spacemacs/prefix-titles)
+      (pcase dotspacemacs-which-key-position
+        ('right (which-key-setup-side-window-right))
+        ('bottom (which-key-setup-side-window-bottom))
+        ('right-then-bottom (which-key-setup-side-window-right-bottom)))
```

Comment:
@mbertheau I didn't understand your comment, because I had tested it in Emacs 25 and there was no problem. I also didn't think it would make any sense to distinguish the two cases, and it looks like pcase was updated between your version and mine to not make a distinction. 

I'll fix this now.